gitops/overlays/olivier/configs/oauth-proxy.conf

```diff
@@ -0,0 +1,42 @@
+## OAuth Proxy Config File
+## https://github.com/oauth2-proxy/oauth2-proxy
+
+## <addr>:<port> to listen on for HTTP/HTTPS clients
+http_address = "0.0.0.0:4180"
+
+## the url root path that this proxy should be nested under
+proxy_prefix = "/oauth"
+
+## the http url(s) of the upstream endpoint
+upstreams = [ "http://localhost:3000/" ]
+
+## email domains to allow authentication for
+email_domains = [ "*" ]
+
+## bypass authentication for requests that match the method & path
+skip_auth_routes = [ "GET=^/api/livez", "GET=^/api/readyz" ]
+
+## mark paths as API routes to get HTTP Status code 401 instead of redirect to login page
+api_routes = [ "^/api" ]
+
+## cookie settings
+cookie_domains = [ "cdcp-dev.dev-dp.dts-stn.com" ]
```

Comment:
This needs to be `cdcp-olivier.dev-dp.dts-stn.com`

Reply:
done